### PR TITLE
Changes in README.md

### LINKED ISSUE #1: Readme badges
Adding some informative badges into README.md

such as: 
[](https://pypi.org/project/squiral/) [](https://pypi.org/project/squiral/)

### LINKED ISSUE #2: Basic documentation
Adding basic documentation (such as install, usage, etc) into README.md

### PR DESCRIPTION
**Pull request by nikitamahoviya**
Fixes #1  and Fixes #2 

I have made changes in the README file which suited the best, may you suggest me any more further changes.

### CODE REVIEW

**Comment by sadikkuzu** [comment]
Thank you so much for your first contribution! 
Without people like you submitting PRs, open source projects couldn't be run. 
You rock, @nikitamahoviya! :rocket:

File: README.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
-# squiral
+![PyPI](https://img.shields.io/pypi/v/pip) [![made-with-python](https://img.shields.io/badge/Made%20with-Python-1f425f.svg)](https://www.python.org/) [![PyPI pyversions](https://img.shields.io/pypi/pyversions/ansicolortags.svg)](https://pypi.python.org/pypi/ansicolortags/)
```
Comment (sadikkuzu) [inline README.md:1]: Actually, this project doesn't support Python 2.7.
We can get data directly from pypi, like those badges:
[ ![ ] ( https://img.shields.io/pypi/v/squiral )] (https://pypi.org/project/squiral/) 
[ ![ ] ( https://img.shields.io/pypi/pyversions/squiral.svg )] (https://pypi.org/project/squiral/)

File: README.md [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
 
-**squ**are sp**iral**
+# Squiral
```
Comment (sadikkuzu) [inline README.md:3]: I prefer title as lowercase. (squiral)

File: README.md [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
+>>> sq.printout(sq.produce(5))
+21 22 23 24 25
+20  7  8  9 10
+19  6  1  2 11
+18  5  4  3 12
+17 16 15 14 13
+>>>
+```
+ 
+**Squ**are sp**iral**
```
Comment (sadikkuzu) [inline README.md:20]: I prefer header as lowercase. (square spiral)

File: README.md [hunk 4]
```diff
@@ -10,3 +27,7 @@ Here is an example:
 19  6  1  2 11
 18  5  4  3 12
 17 16 15 14 13
+```
+The basic idea behind printing this matrix is to start from the middle of the matrix and then moving
```
Comment (sadikkuzu) [inline README.md:31]: Cool description. Thanks a lot! ✨

**Review by sadikkuzu — CHANGES_REQUESTED** [review]
I've added my review as inline comments.
Thanks again.

**Comment by nikitamahoviya** [comment]
> Thank you so much for your first contribution!
> Without people like you submitting PRs, open source projects couldn't be run.
> You rock, @nikitamahoviya!

Thank you so much for the appreciation, would love to contribute more!!

**Comment by sadikkuzu** [comment]
Will be closed because PR#4 covers this changes.

**Pull request closed without merging**